### PR TITLE
IISc-MILE Tamil ASR

### PR DESCRIPTION
**Pull request by RishabSA**
## What did you change?

Created an end-to-end ASR recipe for the [IISc-MILE Tamil ASR Corpus](https://www.openslr.org/127/). Fully trained the model, with results in README.md.

## Why did you make this change?

Made this recipe for my ESPNet Bootcamp project.

## Is your PR small enough?

Yes

---

## Additional Context

I am going to re-train with use_lm, speed perturbation, and some hyperparameter tuning to improve results. Once this model is trained, I will open a new PR and upload it to Hugging Face under the ESPNet organization.

### CODE REVIEW

File: egs2/ta_openslr127/asr1/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+- date: `Sat Jun  6 15:09:08 CDT 2026`
+- python version: `3.10.20 (main, Mar 11 2026, 17:46:40) [GCC 14.3.0]`
+- espnet2 version: `espnet2 202604`
+- pytorch version: `pytorch 2.9.1+cu128`
+- Git hash: `a07484355a508bf041576a37369f7644e4f38497`
+  - Commit date: `Thu Jun 4 22:44:34 2026 -0400`
+- Pretrained model:
+
+## exp/asr_train_asr_raw_ta_bpe1000
```
Comment (sw005320) [inline egs2/ta_openslr127/asr1/README.md]: what is the difference between `exp/asr_train_asr_raw_ta_bpe1000` and `exp/asr_train_asr_raw_ta_bpe1000/decode_asr_asr_model_valid.acc.ave`?

File: egs2/ta_openslr127/asr1/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+## Environments
+
+- date: `Sat Jun  6 15:09:08 CDT 2026`
+- python version: `3.10.20 (main, Mar 11 2026, 17:46:40) [GCC 14.3.0]`
+- espnet2 version: `espnet2 202604`
+- pytorch version: `pytorch 2.9.1+cu128`
+- Git hash: `a07484355a508bf041576a37369f7644e4f38497`
+  - Commit date: `Thu Jun 4 22:44:34 2026 -0400`
+- Pretrained model:
```
Comment (sw005320) [inline egs2/ta_openslr127/asr1/README.md]: Please add an HF model link.

**Review by sw005320 — COMMENTED** [review]
The PR looks good to me overall.
I have minor comments.

File: egs2/ta_openslr127/asr1/README.md (hunk 1 above)
Comment (RishabSA) [reply egs2/ta_openslr127/asr1/README.md]: That part was auto-generated, but the first one is the results for the test set, and the second is one is the results for the validation set.

File: egs2/ta_openslr127/asr1/README.md (hunk 2 above)
Comment (RishabSA) [reply egs2/ta_openslr127/asr1/README.md]: Yep. I retrained with LM and speed perturbation and uploaded it to HuggingFace: [https://huggingface.co/espnet/ta_openslr127](https://huggingface.co/espnet/ta_openslr127). I updated and am pushing the new changes.

**Comment by sw005320** [comment]
Thanks, @RishabSA!
